pyfiglet/__init__.py

```diff
@@ -39,14 +39,22 @@
 
 DEFAULT_FONT = 'standard'
 
+COLORS = {'BLACK': 30, 'RED': 31, 'GREEN': 32, 'YELLOW': 33, 'BLUE': 34, 'MAGENT': 35, 'CYAN': 36, 'LIGHT_GRAY': 37,
+          'DEFAULT': 39, 'DARK_GRAY': 90, 'LIGHT_RED': 91, 'LIGHT_GREEN': 92, 'LIGHT_YELLOW': 93, 'LIGHT_BLUE': 94,
+          'LIGHT_MAGENTA': 95, 'LIGHT_CYAN': 96, 'WHITE': 97
+}
```

Comment:
`MAGENTA` has a typo. Also wonder if it would be good to allow lower case color names to be specified, too, e.g, `--color=red` looks nicer to me than `--color=RED`.

Also, it's possible to have more colours: https://en.wikipedia.org/wiki/ANSI_escape_code#24-bit

Do you want to support those? Easiest way in that way might be to not parse the code but instead let the user pass in an ANSI code, perhaps.

Reply:
Missed that typo, I think I managed to handle 24 bit colors